Examples/tcl/multimap/example.c

```diff
@@ -3,6 +3,12 @@
 #include <stdlib.h>
 #include <ctype.h>
 
+#include <tcl.h>
+
+#ifndef TCL_SIZE_MAX
+typedef int Tcl_Size;
+#endif
+
```

Comment:
This part doesn't seem right - these changes should really only affect the generated code, but here the API being being wrapped is effectively being changed.

Reply:
Here I was too smart in replacing int's with Tcl_Size. 
Should be OK now.